docs/01-overview/02-main-areas/service-management/smgt-14-hb-create-addons-repo.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Create addons repository
```

Comment:
same here, are these instructions that could be presented as a sequence of steps?

Reply:
Changed to tutorial with the same title